src/frontend/src/common/routes.ts

```diff
@@ -22,7 +22,7 @@ export enum ROUTES {
   PANGOLIN = "https://pangolin.cog-uk.io/",
   PHYLO_TREES = "/data/phylogenetic_trees",
   PRIVACY = "/privacy",
-  PRIVACY_PREVIEW = "/privacy-preview",
+  PRIVACY_DATA_COLLECTION = "/privacy#privacy-data-collection",
```

Comment:
I don't see this being used anywhere else in this PR: should it be? Or is the `#` fragment there for external links, like if we have an email linking the change we want it to go directly there?

Reply:
Yeah, I just swapped the files but github tracks the changes as new. The link isn't used anywhere else in the PR, that's correct. It's there for external links, for now. There might be a future use case for using it in the React app. I'm not sure exactly how it will be shared, it has something to do with California privacy law.